examples/pedersen_commitment.rs

```diff
@@ -1,7 +1,5 @@
-use curv::arithmetic::{traits::*, BigInt};
-use curv::elliptic::curves::traits::ECPoint;
-
-use std::fmt::Debug;
+use curv::arithmetic::*;
+use curv::elliptic::curves::*;
 
 /// Pedesen Commitment:
```

Comment:
just noticed the typo (should be Pedersen)

Reply:
Fixed!